guides/drivers.rst

```diff
@@ -20,4 +20,34 @@ Mink comes with five drivers out of the box:
     /drivers/sahi
     /drivers/selenium
 
-.. todo:: Build a table of the features supported by each driver
+.. _driver-feature-support:
+
+Driver Feature Support
+----------------------
+
+Although Mink does its best on removing browser differences between different
+browser emulators, it can't do much in some cases. For example, BrowserKitDriver
+cannot evaluate JavaScript and Selenium2Driver cannot get the response status
+code. In such cases, the driver will always throw a meaningful
+``Behat\Mink\Exception\UnsupportedDriverActionException``.
+
+======================  =================  =========  ======  ========  ====
+Feature                 BrowserKit/Goutte  Selenium2  Zombie  Selenium  Sahi
```

Comment:
I used approximately the same order than when listing them in the toctree (Sahi and Selenium have been switched for no real reason). And the order in the toctree is indeed meaningful. I decided to put the working drivers first (Goutte, Selenium2 and Zombie) order with the pure PHP libraries at the beginning (no deps on Java or Node) and Selenium2 before Zombie because of popularity and then keeping Selenium and Sahi at the end because they have not been released as stable for the Mink 1.6 version and are not passing the testsuite properly